### ISSUE TITLE
[BUG] - ShowHiddenChannels

### ISSUE THREAD

**Issue by Aida-Enna** [2l47/mwittrien-BetterDiscordAddons#1]
**Describe the bug**
No hidden channels are shown for any server

**Additional context**
I am using the the replacement [BDFDB library](https://raw.githubusercontent.com/2l47/mwittrien-BetterDiscordAddons/preserve-ShowHiddenChannels/Library/0BDFDB.plugin.js) from this repository and [the plugin itself from this repo](https://raw.githubusercontent.com/2l47/mwittrien-BetterDiscordAddons/preserve-ShowHiddenChannels/Plugins/ShowHiddenChannels/ShowHiddenChannels.plugin.js).

**Comment by 2l47** [comment]
The original version of the plugin, as written, appears to be non-functional after recent Discord/BetterDiscord updates. I just tested @Kaiddd [Kaiddd's](https://github.com/Kaiddd/ShowHiddenChannels) version, which also appears broken. Until BetterDiscord and other plugin developers sort out their collective non-working plugins, I probably won't be working on this. When things are more stable I may take a look, but otherwise I have to consider the plugin unsupported at this time, as nearly half of the plugins I have installed are broken after BetterDiscord updates.
